libafl_libfuzzer/libafl_libfuzzer_runtime/src/lib.rs

```diff
@@ -27,25 +27,25 @@
     clippy::unsafe_derive_deserialize
 )]
 #![cfg_attr(not(test), warn(
-missing_debug_implementations,
-missing_docs,
-//trivial_casts,
-trivial_numeric_casts,
-unused_extern_crates,
-unused_import_braces,
-unused_qualifications,
-//unused_results
+    missing_debug_implementations,
+    missing_docs,
+    //trivial_casts,
+    trivial_numeric_casts,
+    unused_extern_crates,
+    unused_import_braces,
+    unused_qualifications,
+    //unused_results
 ))]
 #![cfg_attr(test, deny(
-missing_debug_implementations,
```

Comment:
Is this not checked in CI?

Reply:
Not sure, I thought it was but apparently somehow missed.